data/blog/app-feedback-program.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Electron App Feedback Program
+author: sofianguy
+date: '2018-10-04'
+---
+
+Electron is working on building a streamlined release process and having faster releases. To help with that, we started the App Feedback Program for large-scale Electron apps to test our beta releases and report app-specific issues to the Electron team. We use this program to help us prioritize work and get applications upgraded to the next stable release as soon as possible.
+
+## Who can join?
+Our criteria and expectations for apps joining this program include the following items:
+- Test your app during the beta period for 10,000+ user-hours.
+- Have a single point-person to speak for the app's Electron bugs, app blockers, and check-in weekly, either during our regularly scheduled online meeting, or asynchronously with @sofianguy
+- You are willing to share the following information listed in the next question.
+
+## What info does my Electron app have to share?
+- Total user-hours your app has been running with any beta release.
+- Version of Electron that your app is testing with (e.g., 3.0.0-beta.3)
```

Comment:
Might as well bump this to `4.0.0-beta.3` now, or to futureproof it `X.0.0-beta.3` :smile: